aitinero/convex/schema.ts

```diff
@@ -15,12 +15,13 @@ export default defineSchema({
         events: v.optional(v.array(v.id("events")))
     }),
     events: defineTable({
+        title: v.string(),
         duration: v.string(), // in miliseconds from Unix Epoch
         start_time: v.string(),
         end_time: v.string(),
-        location: v.string(),
+        location: v.optional(v.string()),
         status: v.string(), // convert to enum
```

Comment:
Maybe update the typing here.

```suggestion
        status: v.union(v.literal(EVENT_STATUS.Confirmed), v.literal(EVENT_STATUS.Possible)),
```